events_task/lib/my_app_web/templates/event/form.html.eex

```diff
@@ -0,0 +1,19 @@
+<%= form_for @changeset, @action, fn f -> %>
+  <%= if @changeset.action do %>
+    <div class="alert alert-danger">
+      <p>Oops, something went wrong! Please check the errors below.</p>
+    </div>
+  <% end %>
+
+  <%= label f, :description %>
+  <%= text_input f, :description %>
+  <%= error_tag f, :description %>
+<br>
+  <%= label f, :timestamp %>
+  <%= datetime_select f, :timestamp %>
+  <%= error_tag f, :timestamp %>
+  <%= hidden_input f,:author_id, value: @current_user.id %>
```

Comment:
This works for regular uses, but this is a major security issue - value in this field is not protected at all.
As a user, I can replace value of this hidden input to any id, post form - and I will create an event with another user as author.
To fix this, `author_id` should not be set on form - it should be set in controller.